app/views/groups/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<%# <h2>New Category</h2> %>
+
+<%= render partial: 'groups/shared/form', locals: { group:@group }%>
+
+
+
+<%# <%= link_to 'Back', user_groups_path(user_id: current_user.id) %> 
```

Comment:
- [ ] Please kindly delete the commented lines of code to make your codebase clean and professional 🙃